### DISCUSSION TITLE
setting options for SRVPGMs

### DISCUSSION THREAD

**Discussion by JDubbTX** [IBM/ibmi-tobi#174] [Q&A] [answered]
We have srvpgm that requires OPTION(*DUPPROC) on the CRTSRVPGM command.  I've read the documentation and tried this in the make file:

![image](https://user-images.githubusercontent.com/56802156/203629718-91a0b83d-a784-40f6-956f-1f707b7e2a1f.png)

but as shown, the option isn't added to the build command.  What is the proper way to do this?

**Comment by JDubbTX** [comment, marked as answer]
I think I answered my own question.  looking through variables in the SRVPGM recipes, loops like OPTION isn't supported for SRVPGM objects.  I'll try the ILESRVPGM way.

Edit:  While not ideal, putting the CREATE command I want in a source member in ILESRVPGM does seem to work.  I'd much rather be able to do it with just the binder source and not need the extra command source.

**Comment by edmundreinhardt** [comment]
I agree that creating from binder source is cleaner, please create an issue requesting that CRTSRVPGMFLAGS supports the OPTION flag as your require.
